iModelCore/ECDb/ECDb/ECSql/ECSqlPreparer.cpp

```diff
@@ -1857,8 +1863,14 @@ ECSqlStatus ECSqlExpPreparer::PrepareExtractInstanceExp(NativeSqlBuilder::List&
     NativeSqlBuilder builder;
     NativeSqlBuilder::List classIdSql;
     NativeSqlBuilder::List instanceIdSql;
-    if (!ctx.GetECDb().GetECSqlConfig().GetExperimentalFeaturesEnabled()) {
-        ctx.Issues().ReportV(IssueSeverity::Error, IssueCategory::BusinessProperties, IssueType::ECSQL, "Instance access '%s' is experimental feature. Use 'PRAGMA experimental_features_enabled=true' to enable it.", exp.ToECSql().c_str());
+    
+    // Check if ECSQLOption ENABLE_EXPERIMENTAL_FEATURES has been added
+    auto experimentalFeaturesECSqlOption = false;

```

Comment:
Better move this block into a separate method, it's redundant

Reply:
Done !